libraries/AP_HAL_ChibiOS/hwdef/CubeBlack/defaults.parm

```diff
@@ -1,7 +1,23 @@
 # setup correct defaults for battery monitoring for cube power brick
+# default to correct parameters for IMU redundancy 
+# default to ADSB-in on Serial 5
+# disable inflight reboot till more testing is completed
+ADSB_ENABLE 1
+AVD_ENABLE 1
+AVD_F_ACTION 2
+AVD_W_ACTION 1
 BATT2_CURR_PIN 14
 BATT2_VOLT_PIN 13
 BATT_AMP_PERVLT 39.877
 BATT_VOLT_MULT 12.02
 BATT2_AMP_PERVLT 39.877
 BATT2_VOLT_MULT 12.02
+BRD_OPTIONS 0
+BRD_TYPE 3
+EK2_IMU_MASK 7
```

Comment:
that chews an extra 20k of ram. Some boards will no longer boot properly
What are you trying to achieve with this anyway? We already fallback to the 3rd IMU if first two fails.
With this change it would be using a lane that is not logged, and could switch to a non-isolated IMU.
This change will cause crashes